packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -33,7 +33,8 @@ export interface GraphItemLite {
  * @returns Promise which resolves to share link url when successful; otherwise, undefined.
  */
 export async function getShareLink(
-    getShareLinkToken: (options: TokenFetchOptions, isForFileDefaultUrl: boolean) => Promise<string | null>,
+    getShareLinkToken:
+        (options: TokenFetchOptions, isForFileDefaultUrl: boolean, siteUrl: string) => Promise<string | null>,
```

Comment:
I'm not sure a boolean is the right API here, what if we need to add another one in the future? Would a string union be a better fit here?

Long term, I think we should probably move to the TokenProvider model used in office-web-host and move all the scope business into the driver.

Reply:
Any chance we can rename this arg to something more readable? I can't figure out it's meaning :(

Reply:
Added enum for this.